### PR TITLE
added vulcan-config-builder

### PR DESCRIPTION
**Pull request by martingartonft**
Added vulcan-config-builder service. Currently nothing uses it.

### CODE REVIEW

File: vulcan-config-builder.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Vulcand configuration builder.
+Requires=docker.socket
```
Comment (Zapadlo) [inline vulcan-config-builder.service]: I would add etcd2.service and vulcan.service here.

**Comment by Zapadlo** [comment]
+1

File: vulcan-config-builder.service (hunk 1 above)
Comment (martingartonft) [inline vulcan-config-builder.service]: vcb doesn't need vulcan running to set the values in etcd so vulcan can start after vcb just fine.

Comment (Zapadlo) [inline vulcan-config-builder.service]: OK, nvm

**Pull request closed without merging**